app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -23,17 +24,24 @@ const TOP_BAR_HEIGHT = 32
 const props = defineProps<{
   position: Vec2
   navigator: ReturnType<typeof useNavigator>
+  initialContent: string
+  initialCaretPosition: ContentRange
 }>()
 
 const emit = defineEmits<{
-  (e: 'finished'): void
+  finished: [String]
 }>()
 
 onMounted(() => {
-  if (inputField.value != null) {
-    inputField.value.focus({ preventScroll: true })
-    selectLastAfterRefresh()
-  }
+  input.code.value = props.initialContent
+  nextTick(() => {
```

Comment:
Is the next tick really necessary here? It's already after mount, so the DOM node should be present anyway.